ScoutSuite/providers/gcp/resources/cloudresourcemanager/service.py

```diff
@@ -0,0 +1,13 @@
+from ScoutSuite.providers.gcp.facade.gcp import GCPFacade
+from ScoutSuite.providers.gcp.resources.projects import Projects
+from ScoutSuite.providers.gcp.resources.cloudresourcemanager.bindings import Bindings
+
+class CloudResourceManager(Projects):
+    _children = [  
+        (Bindings, 'bindings') 
+    ]
+
+    def __init__(self, gcp_facade: GCPFacade):
+        super(CloudResourceManager, self).__init__(gcp_facade)
```

Comment:
Is this constructor necessary? It's the same as the base.

Reply:
Yeah, after some additional refactoring it became obsolete. Good catch!